### ISSUE TITLE
The "break" position in "repeat/loop" is not taken in account

### ISSUE THREAD

**Issue by StephaneIFO** [Proektsoftbg/Calcpad#79]
Hi,

As said [here](https://calcpad.eu/help/36/iteration-blocks) there can be code before and after the "break" but the code after the "break" is still executed.
 
Test code : 
```
a = 1
b = 1
#repeat
  '-------------
  'a++
  a = a + 1
  
  'test
  #if a ≥ 2
  
    'break
    #break
  #end if
  
  'b++
  b = b + 1
#loop
'##########
a
b
```

Result : 
```
a = 1
b = 1
-------------
a++
a = a + 1 = 1 + 1 = 2

test


break


b++                        <------ should not exist 
b = b + 1 = 1 + 1 = 2      <------ should not exist 

##########
a = 2
b = 2                      <------ must be b = 1   

```

**Comment by Proektsoftbg** [comment]
Thank you for pointing this out to us. It is not expected to execute the code after `#break` and exit on `#loop`. It is expected to exit immediately on `#break`, so this is a serious issue. It is interesting how no one, including me, have noticed that so long. However, I fixed it in the last release v5.8.8:

![image](https://user-images.githubusercontent.com/90918363/195019818-4bc12ef0-37a1-4ad8-a4c8-aeed3e15907e.png)

If you have installed it already, unistall it, download it again and make a clean install to get it work.

**Comment by StephaneIFO** [comment]
I reinstalled v5.8.8 and now it works !
Your software is great, glad to help improve it.
Thank you.
